eogrow/pipelines/switch_grids.py

```diff
@@ -0,0 +1,237 @@
+"""
+Pipelines for testing
+"""
+import logging
+from typing import Any, Dict, List, Optional, Tuple
+
+from pydantic import Field, root_validator
+
+from eolearn.core import EONode, EOWorkflow, FeatureType, LoadTask, OverwritePermission, SaveTask
+from sentinelhub import BBox
+
+from ..core.area.base import AreaManager
+from ..core.eopatch import EOPatchManager
+from ..core.pipeline import Pipeline
+from ..core.schemas import BaseSchema, ManagerSchema
+from ..tasks.spatial import SpatialJoinTask, SpatialSliceTask
+from ..utils.grid import GridTransformation
+from ..utils.types import Feature, FeatureSpec
+from ..utils.validators import field_validator, optional_field_validator, validate_manager
+
+LOGGER = logging.getLogger(__name__)
+
+
+class FeatureSchema(BaseSchema):
+    feature_type: FeatureType
+    feature_name: Optional[str] = Field(description="A feature name if it exists")
```

Comment:
why is feature name optional?

Reply:
So that also timestamp feature type could be specified here. Otherwise, we would need an additional parameter for timestamp propagation. :thinking:

Reply:
I'm slowly gravitating towards having flags for bbox and timestamp, makes working with other features a breeze. But perhaps outside of scope for this MR. Why not use `FeatureSpec` and have a single param?

Reply:
For now I changed it to a single parameter so that it will be inline with other pipelines.